src/idl_gen_kotlin.cpp

```diff
@@ -140,11 +140,11 @@ class KotlinGenerator : public BaseGenerator {
   static std::string GenTypeBasic(const BaseType &type) {
     // clang-format off
     static const char * const kotlin_typename[] = {
-      #define FLATBUFFERS_TD(ENUM, IDLTYPE, \
```

Comment:
This file contains lots of unrelated formatting changes, can you undo them all?

Reply:
I'll do that soon.

Reply:
@aardappel I undo all unrelated formatting change.